src/qt/forms/optionsdialog.ui

```diff
@@ -507,10 +507,10 @@
        <item>
         <widget class="QCheckBox" name="hideTrayIcon">
          <property name="toolTip">
-          <string>&amp;Hide the icon from the system tray.</string>
+          <string>Hide the icon from the system tray.</string>
          </property>
          <property name="text">
-          <string>Hide tray icon</string>
+          <string>&amp;Hide tray icon</string>
```

Comment:
Redundant with auto-assignment. Suggest leaving it alone until 0.16. (But still fix the tooltip)